core/assets/exim/25_mm3_macros

```diff
@@ -2,7 +2,7 @@
 # /etc/exim4/conf.d/main/25_mm3_macros
 
 domainlist mm3_domains=MY_DOMAIN_NAME
-MM3_LMTP_HOST=172.19.199.2
+MM3_LMTP_HOST=mailman-core
```

Comment:
This is the most problematic aspect of actually removing the IP addresses. Host doesn't have the container-name -> IP mapping in its own '/etc/hosts' file, which would make it impossible for containers on the host to discover this. 

However, I think a better idea is to use local port mapping (127.0.0.1:8024 from containers 8024) instead so that Web Server and Nginx can route requests to containers without knowledge of the container IP addresses. I should've done that from the start instead of this static ip nonsense.

Reply:
This change works for me because my Exim instance is dockerized as well, and shares the same bridge with `mailman-core`.

Each end user's configuration is different and will require a specific network setup, depending on whether they use a reverse proxy for example. Then for this file I think we might just use a placeholder, as for the domain name:
```
MM3_LMTP_HOST=MAILMAN_CORE_HOSTNAME_OR_IP
```

Reply:
With port mapping in `docker-compose.yml` it should work with `localhost` if the MTA is on the host. See 0284ab4.